src/protocol/requests/deleteRecords/v1/response.js

```diff
@@ -0,0 +1,13 @@
+const responseV0 = require('../v0/response')
+
+/**
+ * DeleteRecords Response (Version: 1) => throttle_time_ms [topics]
+ *  throttle_time_ms => INT32
+ *  topics => name [partitions]
+ *    name => STRING
+ *    partitions => partition_index low_watermark error_code
+ *      partition_index => INT32
+ *      low_watermark => INT64
+ *      error_code => INT16
+ */
+module.exports = responseV0
```

Comment:
In V1 the broker is returning the response before throttling the client in case the client has exceeded the quota. This means that we should be throttling the client on the client side, instead of relying on the server to do it for us.

See [this comment](https://github.com/tulios/kafkajs/pull/697#discussion_r498360020) for more information and an example. It's a very simple change.